text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/parser/TokenParser.java

```diff
@@ -156,13 +166,13 @@ public static void parseString(final String message, final MatchedTokenConsumer<
 
       if (!escaped) {
         // if we're trying to escape and the next character exists
-        if (codePoint == Tokens.ESCAPE && i + 1 < message.length()) {
+        if (codePoint == TokenParser.ESCAPE && i + 1 < message.length()) {
```

Comment:
Do these have to be qualified with `TokenParser.` ? I had assumed the "closer to where they are used" in #637 was to avoid smth like this

Reply:
they do not, this is just eclipse's move button not cleaning up :p